revproxy/utils.py

```diff
@@ -3,31 +3,42 @@
 
 import logging
 
-
+#: List containing string constant that are used to represent headers that can
+#: be ignored in the required_header function
 IGNORE_HEADERS = (
     'HTTP_ACCEPT_ENCODING',  # We want content to be uncompressed so
-                             #   we remove the Accept-Encoding from
-                             #   original request
+                             # we remove the Accept-Encoding from
+                             # original request
     'HTTP_HOST',
 )
 
 
 # Default from HTTP RFC 2616
 #   See: http://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.7.1
+#: Variable that represent the default charset used
 DEFAULT_CHARSET = 'latin-1'
 
+#: List containing string constants that represents possible html content type
```

Comment:
List containing strings is also implicit here. That's a constant so we don't have to document input types or anything like that.

Those are the content types used to determine whether a response is HTML or not.